source/time_stepping/repeat_on_cutback.cc

```diff
@@ -0,0 +1,122 @@
+/*
+  Copyright (C) 2018 - 2020 by the authors of the ASPECT code.
+
+  This file is part of ASPECT.
+
+  ASPECT is free software; you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation; either version 2, or (at your option)
+  any later version.
+
+  ASPECT is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with ASPECT; see the file LICENSE.  If not see
+  <http://www.gnu.org/licenses/>.
+*/
+
+
+#include <aspect/global.h>
+#include <aspect/time_stepping/repeat_on_cutback.h>
+
+namespace aspect
+{
+  namespace TimeStepping
+  {
+    template <int dim>
+    double
+    RepeatOnCutback<dim>::execute()
+    {
+      return std::numeric_limits<double>::max();
+    }
+
+
+
+    template <int dim>
+    std::pair<Reaction, double>
+    RepeatOnCutback<dim>::determine_reaction (const TimeStepInfo &info)
+    {
+      std::cout
+          << "RepeatOnCutback old:" << this->get_old_timestep()
+          << " last: " << this->get_timestep()
+          << " next: " << info.next_time_step_size
+          << " last? " << info.reduced_by_termination_plugin
+          << std::endl;
```

Comment:
Is this debug output? If so remove. If not it would need a bit more explanation, like `computed timestep:`, `modified timestep:`...